cfgov/jinja2/v1/complaint/complaint-landing.html

```diff
@@ -68,22 +68,16 @@ <h1 class="m-hero_heading">Consumer Complaint Database</h1>
 
 {% block content_main %}
 
-<div class="m-notification m-notification__warning {% if data_down %} m-notification__visible show-data-notification {% elif narratives_down %} m-notification__visible show-narratives-notification {% endif %}">
+<div class="m-notification m-notification__warning {% if technical_issues %} m-notification__visible show-data-notification {% endif %}">
```

Comment:
If this is a straight-up static notification that is shown/hidden it looks like we can get rid of the `show-data-notification` class here, `data-text` class on line 76, and this CSS https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/ccdb-landing-map/css/landing-page.less#L63-L80

Reply:
√